### PR TITLE
backend for the map

### PR DESCRIPTION
**Pull request by ThaiBangHOANG**
Hello,
I have just added the database for the map part on the backend.
Thank you very much for your verification and validation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,10 @@
     "cors": "^2.8.5",
     "dotenv": "^9.0.2",
     "express": "^4.17.1",
-    "mysql2": "^2.2.5"
+    "mysql2": "^2.2.5",
+    "react": "^17.0.2",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0"
```
Comment (GeorgesFORTRY) [inline package.json]: useless in backend

**Review by GeorgesFORTRY — APPROVED** [review]
OK but no react in backend

File: package.json (hunk 1 above)
Comment (LoicBrassart) [inline package.json]: Totally, yep
So I wonder why you validated the PR, @GeorgesFORTRY ? :p

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ThaiBangHOANG) [reply package.json]: I did not know why react has installed in the backend as well :p

**Review by GeorgesFORTRY — APPROVED** [review]
ok 😥

**Review by LoicBrassart — APPROVED** [review]
Much better

**Pull request merged**